### ISSUE TITLE
[Feature Request]: Disk cache entry should be whole S3 object

### ISSUE THREAD

**Issue by fengttt** [matrixorigin/matrixone#11727]
### Is there an existing issue for the same feature request?

- [X] I have checked the existing issues.

### Is your feature request related to a problem?

```Markdown
We should split metadata cache and data cache.   Here we do not discuss metadata cache.

For data cache, in memory we will cache column block (uncompressed).   For disk cache, we should cache the whole S3 object.
```

### Describe the feature you'd like

We should split metadata cache and data cache.   Here we do not discuss metadata cache.

For data cache, in memory we will cache column block (uncompressed).   For disk cache, we should cache the whole S3 object.

### Additional information

I thought we already has such an issue tracking this, but I cannot find it.

Basically, for TPCC, we want at MOST 2 S3 request per S3 object.    Right now we issue at least 2 orders of magnitue more requests.

**Comment by reusee** [comment]
Seems like https://github.com/matrixorigin/matrixone/issues/11569

**Comment by fengttt** [comment]
Close as dup
